.github/workflows/pre-release.yaml

```diff
@@ -33,12 +39,20 @@ jobs:
         with:
           path: /tmp/ucm
 
+      - name: derive release tag
+        run: |
+          if [ -n "${{ github.event.inputs.tag }}" ]; then
+            echo "ref_name=${{ github.event.inputs.tag }}"
+          else
+            echo "ref_name=$(echo ${{ github.ref_name }} | awk -F'/' '{print $NF}')"
```

Comment:
Are you meaning to set env vars here?
At a glance it looks like this just echos it to stdout, but I assume you've tested this somehow?

I didn't see this syntax mentioned in [this guide](https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow) so am just a bit confused how this works 🤔

Reply:
Oh yeah, good catch!
This section had a bunch of iteration, a deletion, and then the final version was written by Copilot and I didn't notice the omission.